.circleci/config.yml

```diff
@@ -1,17 +1,87 @@
+version: 2.1
+
+workflows:
+  version: 2
+  commit:
+    jobs:
+      - build_android:
+          e2e_build: true
+      - build_ios:
+          e2e_build: true
+      - e2e:
+          requires:
+            - build_ios
+            - build_android
+  weekly:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *" # FIXME "0 0 * * 3"
+          filters:
+            branches:
+              only:
+                - develop
```

Comment:
So what about our master branch now? Should we delete it? Or should we make `develop` our `master` branch?

Reply:
We will probably remove it, but we will see how this CD stuff works